image/templates/helm/stackrox-central/templates/00-securitypolicy-crd.yaml

```
@@ -0,0 +1,4 @@
+{{- include "srox.init" . -}}
+{{ if ne (._rox.allowUnsupportedMultipleCentralsUsingHelm | toString) "documentationWasReadAndUnderstood" -}}
+{{ $.Files.Get "internal/config.stackrox.io_securitypolicies.yaml" }}
+{{- end -}}
```

I am advocating for keeping the templating style (especially with respect to whitespace elimination) somewhat consistent in our code base, hence I would like to suggest to follow [our current guide lines](https://github.com/stackrox/stackrox/blob/6ee9603c57f97efa33f272ed01815709b073d6dd/image/templates/CHANGING_CHARTS.md#L5).

```suggestion
{{- if ne (._rox.allowUnsupportedMultipleCentralsUsingHelm | toString) "documentationWasReadAndUnderstood" }}
{{ $.Files.Get "internal/config.stackrox.io_securitypolicies.yaml" }}
{{- end }}
```